tests/e2e/cypress/integration/search.spec.js

```diff
@@ -453,15 +450,15 @@ describe('Search', function() {
 
     cy.get('.RawFilter-submitBtn').click()
 
-    cy.get('.DocumentListItem').should(function($el) {
+    cy.get('.DocumentListItem').should(function ($el) {
       expect($el.first()).to.contain('Maret')
       expect($el.last()).to.contain('Marchesini')
```

Comment:
Shouldn't be like line 386 ?
```suggestion
      expect($el.last()).to.contain('marchesini')
```

Reply:
same as previous